care/emr/migrations/0011_medicationrequest_requester.py

```
@@ -0,0 +1,21 @@
+# Generated by Django 5.1.4 on 2025-01-23 05:04
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emr', '0010_condition_abatement'),
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)